sentry_streams/sentry_streams/examples/simple_map_filter.py

```diff
@@ -19,20 +28,18 @@ def generate_files() -> str:
     return f"file_{cur_time}.txt"
 
 
-# A pipline with a few transformations
-pipeline = (
-    streaming_source(
-        name="myinput",
-        stream_name="ingest-metrics",
-    )
-    .apply(
-        "parser",
-        Parser(
-            msg_type=IngestMetric,
-        ),
+pipeline = streaming_source(name="myinput", stream_name="ingest-metrics")
+
+(
+    pipeline.apply(Parser("parser", msg_type=IngestMetric))
+    .apply(Filter("filter", function=filter_events))
+    .apply(Map("transform", function=transform_msg))
+    .apply(Serializer("serializer"))
+    .sink(
+        GCSSink(
+            "mysink",
+            bucket="arroyo-artifacts",
+            object_generator=generate_files,
+        )
     )
```

Comment:
Could you please confirm whether this change was not required and it is just refactoring the existing code?